### PR TITLE
Feature/separate profile for rcp

### PR DESCRIPTION
**Pull request by gunther-bachmann**

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -65,7 +69,7 @@ task deploy(type: MavenExec) {
 // TODO should be dependent on build (but build needs to be incremental for that)
```
Comment (jjelschen) [inline build.gradle]: why can't it be?

File: build.gradle [hunk 2]
```diff
@@ -51,8 +51,12 @@ task cleanTarget(type: MavenExec) {
     args 'clean'
 }
 
+task assemble() {
+}
+
 // TODO define inputs and outputs
```
Comment (jjelschen) [inline build.gradle]: don't know what that means (might be due to a lack of understanding of Gradle, though…)

File: build.gradle (hunk 1 above)
Comment (gunther-bachmann) [reply build.gradle]: todo is a remaineder of the previous editor :-|

File: build.gradle (hunk 2 above)
Comment (gunther-bachmann) [reply build.gradle]: had to include this empty task to satisfy travis (that sees a gradle build configuration and assumes that this task must be executed for a java project)

**Pull request merged**